### PR TITLE
Upstream Admin UI

### PR DESCRIPTION
**Pull request by ritesh-skyach**
Upstream Admin UI

### CODE REVIEW

File: app/views/admin/upstreams/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<h3>Upstream info</h3>
+
+<%= form_for @upstream, url: @upstream.new_record? ? admin_upstreams_url : admin_upstream_url(@upstream.id) do |f| %>
+  <div class="card-deck float-left col-lg-12">
+    <div class="card col-lg-6 ml-0">
+      <div class="card-body">
+        <div class="form-group">
+          <label>Provider</label>
+          <%= f.text_field :provider, class: 'form-control mb-3' %>
```
Comment (seletskiy) [inline app/views/admin/upstreams/show.html.erb]: Please replace it to selectbox with single possible value — Binance

File: app/views/admin/upstreams/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+          <label>Api Key</label>
```
Comment (seletskiy) [inline app/views/admin/upstreams/show.html.erb]: Please replace `Api` with `API` since it's acronym.

**Comment by seletskiy** [comment]
@ritesh-skyach: we should not allow to enable upstreams in markets with `fiat` currency, can you please add required conditions?

File: app/views/admin/upstreams/show.html.erb (hunk 1 above)
Comment (dinesh-skyach) [reply app/views/admin/upstreams/show.html.erb]: @seletskiy Done

File: app/views/admin/upstreams/show.html.erb (hunk 2 above)
Comment (dinesh-skyach) [reply app/views/admin/upstreams/show.html.erb]: @seletskiy done

**Pull request merged**